packages/markup/markup.0.8.1/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+
+synopsis: "Error-recovering functional HTML5 and XML parsers and writers"
+
+version: "0.8.1"
+license: "BSD"
+homepage: "https://github.com/aantron/markup.ml"
+doc: "http://aantron.github.io/markup.ml"
+bug-reports: "https://github.com/aantron/markup.ml/issues"
+
+authors: "Anton Bachin <antonbachin@yahoo.com>"
+maintainer: "Anton Bachin <antonbachin@yahoo.com>"
+dev-repo: "git+https://github.com/aantron/markup.ml.git"
+
+depends: [
+  "dune" {build}
+  "uchar"
+  "uutf" {>= "1.0.0"}
+
+  "bisect_ppx" {dev & >= "1.3.0"}
+  "ounit" {dev}
```

Comment:
```suggestion
  "ounit" {with-test}
```
Are you sure it's `dev` and not `with-test` that you wanted here?

Reply:
I chose `dev` because on a development pin, `ounit` should be installed. It won't be installed if `with-test` is used.

Reply:
I'm surprised. How do you manage to detect if it's a development pin or a release? How do you use `ounit` for the compilation?

Reply:
I don't know exactly how opam detects this, but compare the dependency cone of `opam install odoc` and `opam pin add odoc --dev-repo`. The extra dependencies are marked with `{dev}` and their transitive dependencies.

I think the issue is that there isn't a good way (AFAIK) to distinguish between a pin for development and a git pin for another reason. So far, I've preferred to overestimate the dependencies for non-development git pins, rather than underestimate dependencies for development.

Reply:
It's not really what I meant but it's ok. It's not really important anyway.